src/main/java/frc/robot/subsystems/shooterSubsystem/ShooterCommandFactory.java

```diff
@@ -82,6 +86,13 @@ public Command shooterIdle() {
         subsystem
             .run(
                 () -> {
+                  if (RSM.getPlacementMode() == PlacementMode.SPEAKER) {
+                    subsystem.setShooterSpeeds(
+                        new SpeakerConfig(
+                            -1,
+                            ShooterConstants.SHOOTER_IDLE_SPEED_RIGHT,
```

Comment:
IDLE isn't the right speed for a speaker shot.  It's showing as 400 RPM

Reply:
This is the idle mode. The check for mode earlier is so we are only idling when we are about to shot